Dryad Digital Repository.js

```diff
@@ -0,0 +1,686 @@
+				"archive": "Dryad Digital Repository",
+				"extra": "{:itemType: dataset}",
+				"libraryCatalog": "datadryad.org",
+				"attachments": [],
+				"tags": [
+					"Biogeography",
+					"Divergence Time Estimation",
+					"Phylogenetics"
+				],
+				"notes": [],
+				"seeAlso": []
+			},
+			{
+				"itemType": "journalArticle",
+				"title": "Data from: Step matrices and the interpretation of homoplasy",
+				"creators": [
+					{
+						"firstName": "Richard H.",
+						"lastName": "Ree",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Michael J.",
+						"lastName": "Donoghue",
+						"creatorType": "author"
+					}
+				],
+				"date": "1998-12-30",
+				"DOI": "10.5061/dryad.119",
+				"abstractNote": "Assumptions about the costs of character change, coded in the form of a step matrix, determine most-parsimonious inferences of character evolution on phylogenies. We present a graphical approach to exploring the relationship between cost assumptions and evolutionary inferences from character data. The number of gains and losses of a binary trait on a phylogeny can be plotted over a range of cost assumptions, to reveal the inflection point at which there is a switch from more gains to more losses and the point at which all changes are inferred to be in one direction or the other. Phylogenetic structure in the data, the tree shape, and the relative frequency of states among the taxa influence the shape of such graphs and complicate the interpretation of possible permutation-based tests for directionality of change. The costs at which the most-parsimonious state of each internal node switches from one state to another can also be quantified by iterative ancestral-state reconstruction over a range of costs. This procedure helps identify the most robust inferences of change in each direction, which should be of use in designing comparative studies.",
+				"archive": "Dryad Digital Repository",
+				"extra": "{:itemType: dataset}",
+				"language": "en_US",
+				"libraryCatalog": "datadryad.org",
+				"attachments": [],
+				"tags": [
+					"ancestral states",
+					"character evolution",
+					"homoplasy",
+					"parsimony",
+					"phylogenetic inference"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
```

Comment:
The tests in the most recent commit were added by Scaffold, and the 'multiples' test does have item data added for each individual item.